index.html

```diff
@@ -5,194 +5,263 @@
-                <li><a href="#"><i class="fa-regular fa-hand-peace fa-flip-horizontal fa-lg"></i></a></li>
-            </ul>
-            <button class="see-projects-btn">See Projects</button>
-            <ul class="catgories">
-                <li class="language">
-                    <span>Languages</span>
-                    <i class='fas fa-angle-down'></i>
+            <div class="lang-fram-skills">
+            <ul class="language">
+                <li class="language-item">
+                    <h3>Languages</h3>
+                <i class="fa fa-chevron-down"></i>
                 </li>
-                <li class="tech js">
-                    <img src="./assets/js.png" alt="">
-                    <span>JavaScript</span>
+            </ul>
+            <ul class="language-list">
+                
+                <li class="css">
+                    <img src="./assets/css.png" alt="">
+                    <span>CSS</span>
                 </li>
-                <li class="tech html">
+
+                <li class="html">
                     <img src="./assets/html.png" alt="">
                     <span>HTML</span>
                 </li>
-                <li class="tech css">
-                    <img src="./assets/css.png" alt="">
-                    <span>CSS</span>
+
+                <li class="js">
+                    <img src="./assets/js.png" alt="">
+                    JavaScript
                 </li>
-                <li class="framework">
-                    <span>Frameworks</span>
-                    <i class='fas fa-angle-right'></i>
+            </ul>
+            <ul class="frameworks">
+                <li class="frameworks-item"><h3>Frameworks</h3>
+                    <i class="fa fa-chevron-right"></i>
                 </li>
-                <li class="skill">
-                    <span>Skills</span>
-                    <i class='fas fa-angle-right'></i>
+            </ul>
+            <ul class="skills">
+                <li class="skills-item"><div><h3>Skills</h3></div>
+                    <div><i class="fa fa-chevron-right"></i></div>
                 </li>
             </ul>
         </div>
     </section>
-    <section class="contact-form">
-        <div class="contact-form-section-wrapper">
-            <div class="text-content">
-                <h2>Contact Me</h2>
-                <p>f you have an application you are interested in
-                developing, a feature that you need built or a project that needs coding. I'd love to help with it!</p>
+    <footer>
+        <section class="footer-section pad-aside">
+            <div class="footer-text">
+                <h1>Contact me</h1>
+                <p>
+                    If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                    I'd love to help with it!
+                </p>
             </div>
```

Comment:
- [ ] Kindly reduce the spacing between the Contact me heading and likewise reduce the horizontal margin of the text beneath 😉.

- Your design
<img width="753" alt="image" src="https://user-images.githubusercontent.com/41163571/236345091-2c7d0885-6e0c-4b66-bd5b-ff903f4390ac.png">

- Figma design
above top